src/components/Auth/Login/Login.js

```diff
@@ -1,9 +1,66 @@
 import React, { Component } from 'react';
+import './sample.scss';
 
-class Login extends Component {
+class sample extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  handleEmailInput = e => {
+    this.setState({
+      email: e.target.value,
+    });
+  };
+  handlePwdInput = e => {
+    this.setState({
+      password: e.target.value,
+    });
+  };
   render() {
```

Comment:
가독성을 위해 각 함수 간에는 한 줄 씩 띄어주시기 바랍니다.
```suggestion
  handleEmailInput = e => {
    this.setState({
      email: e.target.value,
    });
  };
  
  handlePwdInput = e => {
    this.setState({
      password: e.target.value,
    });
  };
  
  render() {
```